data/mods/gen3/scripts.js

```diff
@@ -71,11 +71,11 @@ let BattleScripts = {
 			return false;
 		}
 
-		const targets = pokemon.getMoveTargets(move, target);
+		const {targets, ppTargets} = pokemon.getMoveTargets(move, target);
```

Comment:
I said `pressureTargets` because I think "pp" is unnecessarily vague. How am I supposed to figure it out means `targetsForThePurposesOfCalculatingAdditionalPPDeductionFromEffects`? Pressure is _waaaayyyyy_ clearer.